clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -102,6 +107,7 @@ public Options options() {
     options.addOption(createArgOption(DEFAULT, "default column value"));
     options.addOption(createSimpleOption("o", OWNER, "display entity owner"));
     options.addOption(createArgOption(COLUMNFILE, "CSV file describing columns"));
+    options.addOption(createSimpleOption("R", RECURSIVE, "recursive operation"));
```

Comment:
Looks like my comment on being better names for this was missed?

Reply:
fix